2025-03-09 22:17:30,545 INFO    StreamThr :3912 [internal.py:wandb_internal():86] W&B internal server running at pid: 3912, started at: 2025-03-09 22:17:30.543367
2025-03-09 22:17:30,551 DEBUG   HandlerThread:3912 [handler.py:handle_request():146] handle_request: status
2025-03-09 22:17:30,628 INFO    WriterThread:3912 [datastore.py:open_for_write():85] open: c:\Users\D. Sai Mani Kumar\Downloads\DL\da6401_assignment1\wandb\run-20250309_221730-el3kt5nf\run-el3kt5nf.wandb
2025-03-09 22:17:30,633 DEBUG   SenderThread:3912 [sender.py:send():382] send: header
2025-03-09 22:17:30,759 DEBUG   SenderThread:3912 [sender.py:send():382] send: run
2025-03-09 22:17:31,752 INFO    SenderThread:3912 [dir_watcher.py:__init__():211] watching files in: c:\Users\D. Sai Mani Kumar\Downloads\DL\da6401_assignment1\wandb\run-20250309_221730-el3kt5nf\files
2025-03-09 22:17:31,752 INFO    SenderThread:3912 [sender.py:_start_run_threads():1136] run started: el3kt5nf with start time 1741538850.54437
2025-03-09 22:17:31,771 DEBUG   HandlerThread:3912 [handler.py:handle_request():146] handle_request: check_version
2025-03-09 22:17:31,772 DEBUG   SenderThread:3912 [sender.py:send_request():409] send_request: check_version
2025-03-09 22:17:34,873 DEBUG   HandlerThread:3912 [handler.py:handle_request():146] handle_request: run_start
2025-03-09 22:17:36,171 DEBUG   HandlerThread:3912 [system_info.py:__init__():32] System info init
2025-03-09 22:17:36,172 DEBUG   HandlerThread:3912 [system_info.py:__init__():47] System info init done
2025-03-09 22:17:36,172 INFO    HandlerThread:3912 [system_monitor.py:start():194] Starting system monitor
2025-03-09 22:17:36,173 INFO    SystemMonitor:3912 [system_monitor.py:_start():158] Starting system asset monitoring threads
2025-03-09 22:17:36,173 INFO    HandlerThread:3912 [system_monitor.py:probe():214] Collecting system info
2025-03-09 22:17:36,187 INFO    SystemMonitor:3912 [interfaces.py:start():190] Started cpu monitoring
2025-03-09 22:17:36,189 INFO    SystemMonitor:3912 [interfaces.py:start():190] Started disk monitoring
2025-03-09 22:17:36,191 INFO    SystemMonitor:3912 [interfaces.py:start():190] Started gpu monitoring
2025-03-09 22:17:36,206 INFO    SystemMonitor:3912 [interfaces.py:start():190] Started memory monitoring
2025-03-09 22:17:36,219 DEBUG   HandlerThread:3912 [system_info.py:probe():196] Probing system
2025-03-09 22:17:36,237 INFO    SystemMonitor:3912 [interfaces.py:start():190] Started network monitoring
2025-03-09 22:17:36,260 DEBUG   HandlerThread:3912 [system_info.py:_probe_git():181] Probing git
2025-03-09 22:17:36,519 DEBUG   HandlerThread:3912 [system_info.py:_probe_git():189] Probing git done
2025-03-09 22:17:36,519 DEBUG   HandlerThread:3912 [system_info.py:probe():244] Probing system done
2025-03-09 22:17:36,520 DEBUG   HandlerThread:3912 [system_monitor.py:probe():223] {'os': 'Windows-10-10.0.26100-SP0', 'python': '3.9.5', 'heartbeatAt': '2025-03-09T16:47:36.238371', 'startedAt': '2025-03-09T16:47:30.482228', 'docker': None, 'cuda': None, 'args': (), 'state': 'running', 'program': '<python with no main file>', 'codePathLocal': None, 'git': {'remote': 'git@github.com-work:saimanikumar-da24m016/da6401_assignment1.git', 'commit': '3ddbfd47c7127abba95926229ee5c61c7e5e5eb9'}, 'email': 'saimanikumar67@gmail.com', 'root': 'C:/Users/D. Sai Mani Kumar/Downloads/DL/da6401_assignment1', 'host': 'LAPTOP-VRI27UL2', 'username': 'D. Sai Mani Kumar', 'executable': 'c:\\Users\\D. Sai Mani Kumar\\AppData\\Local\\Programs\\Python\\Python39\\python.exe', 'cpu_count': 4, 'cpu_count_logical': 8, 'cpu_freq': {'current': 1500.0, 'min': 0.0, 'max': 2400.0}, 'cpu_freq_per_core': [{'current': 1500.0, 'min': 0.0, 'max': 2400.0}], 'disk': {'/': {'total': 375.82519149780273, 'used': 261.6537284851074}}, 'gpu': 'NVIDIA GeForce GTX 1650 Ti', 'gpu_count': 1, 'gpu_devices': [{'name': 'NVIDIA GeForce GTX 1650 Ti', 'memory_total': 4294967296}], 'memory': {'total': 7.840869903564453}}
2025-03-09 22:17:36,520 INFO    HandlerThread:3912 [system_monitor.py:probe():224] Finished collecting system info
2025-03-09 22:17:36,520 INFO    HandlerThread:3912 [system_monitor.py:probe():227] Publishing system info
2025-03-09 22:17:36,520 DEBUG   HandlerThread:3912 [system_info.py:_save_pip():52] Saving list of pip packages installed into the current environment
2025-03-09 22:17:36,522 DEBUG   HandlerThread:3912 [system_info.py:_save_pip():68] Saving pip packages done
2025-03-09 22:17:36,524 INFO    HandlerThread:3912 [system_monitor.py:probe():229] Finished publishing system info
2025-03-09 22:17:36,567 DEBUG   HandlerThread:3912 [handler.py:handle_request():146] handle_request: status_report
2025-03-09 22:17:36,568 DEBUG   SenderThread:3912 [sender.py:send():382] send: files
2025-03-09 22:17:36,568 INFO    SenderThread:3912 [sender.py:_save_file():1392] saving file wandb-metadata.json with policy now
2025-03-09 22:17:36,636 DEBUG   HandlerThread:3912 [handler.py:handle_request():146] handle_request: stop_status
2025-03-09 22:17:36,637 DEBUG   SenderThread:3912 [sender.py:send_request():409] send_request: stop_status
2025-03-09 22:17:36,651 DEBUG   HandlerThread:3912 [handler.py:handle_request():146] handle_request: internal_messages
2025-03-09 22:17:36,813 INFO    Thread-16 :3912 [dir_watcher.py:_on_file_created():271] file/dir created: c:\Users\D. Sai Mani Kumar\Downloads\DL\da6401_assignment1\wandb\run-20250309_221730-el3kt5nf\files\wandb-metadata.json
2025-03-09 22:17:36,815 INFO    Thread-16 :3912 [dir_watcher.py:_on_file_created():271] file/dir created: c:\Users\D. Sai Mani Kumar\Downloads\DL\da6401_assignment1\wandb\run-20250309_221730-el3kt5nf\files\requirements.txt
2025-03-09 22:17:37,356 DEBUG   SenderThread:3912 [sender.py:send():382] send: telemetry
2025-03-09 22:17:37,356 DEBUG   SenderThread:3912 [sender.py:send():382] send: telemetry
2025-03-09 22:17:37,356 DEBUG   SenderThread:3912 [sender.py:send():382] send: run
2025-03-09 22:17:37,670 INFO    SenderThread:3912 [sender.py:send_run():1010] updated run: el3kt5nf
2025-03-09 22:17:37,681 INFO    wandb-upload_0:3912 [upload_job.py:push():131] Uploaded file C:\Users\D9110~1.SAI\AppData\Local\Temp\tmp3ujfop86wandb\vdyyfq9k-wandb-metadata.json
2025-03-09 22:17:37,814 INFO    Thread-16 :3912 [dir_watcher.py:_on_file_modified():288] file/dir modified: c:\Users\D. Sai Mani Kumar\Downloads\DL\da6401_assignment1\wandb\run-20250309_221730-el3kt5nf\files\config.yaml
2025-03-09 22:17:37,815 INFO    Thread-16 :3912 [dir_watcher.py:_on_file_created():271] file/dir created: c:\Users\D. Sai Mani Kumar\Downloads\DL\da6401_assignment1\wandb\run-20250309_221730-el3kt5nf\files\output.log
2025-03-09 22:17:38,328 ERROR   gpu       :3912 [interfaces.py:monitor():144] Failed to sample metric: Unknown Error
2025-03-09 22:17:38,329 ERROR   gpu       :3912 [interfaces.py:monitor():144] Failed to sample metric: Unknown Error
2025-03-09 22:17:39,837 INFO    Thread-16 :3912 [dir_watcher.py:_on_file_modified():288] file/dir modified: c:\Users\D. Sai Mani Kumar\Downloads\DL\da6401_assignment1\wandb\run-20250309_221730-el3kt5nf\files\output.log
2025-03-09 22:17:40,769 ERROR   gpu       :3912 [interfaces.py:monitor():144] Failed to sample metric: Unknown Error
2025-03-09 22:17:40,771 ERROR   gpu       :3912 [interfaces.py:monitor():144] Failed to sample metric: Unknown Error
2025-03-09 22:17:41,717 DEBUG   HandlerThread:3912 [handler.py:handle_request():146] handle_request: status_report
2025-03-09 22:17:43,205 ERROR   gpu       :3912 [interfaces.py:monitor():144] Failed to sample metric: Unknown Error
2025-03-09 22:17:43,206 ERROR   gpu       :3912 [interfaces.py:monitor():144] Failed to sample metric: Unknown Error
2025-03-09 22:17:45,679 ERROR   gpu       :3912 [interfaces.py:monitor():144] Failed to sample metric: Unknown Error
2025-03-09 22:17:45,680 ERROR   gpu       :3912 [interfaces.py:monitor():144] Failed to sample metric: Unknown Error
2025-03-09 22:17:46,771 DEBUG   HandlerThread:3912 [handler.py:handle_request():146] handle_request: status_report
2025-03-09 22:17:48,134 ERROR   gpu       :3912 [interfaces.py:monitor():144] Failed to sample metric: Unknown Error
2025-03-09 22:17:48,135 ERROR   gpu       :3912 [interfaces.py:monitor():144] Failed to sample metric: Unknown Error
2025-03-09 22:17:50,666 ERROR   gpu       :3912 [interfaces.py:monitor():144] Failed to sample metric: Unknown Error
2025-03-09 22:17:50,667 ERROR   gpu       :3912 [interfaces.py:monitor():144] Failed to sample metric: Unknown Error
2025-03-09 22:17:51,646 DEBUG   HandlerThread:3912 [handler.py:handle_request():146] handle_request: internal_messages
2025-03-09 22:17:51,646 DEBUG   HandlerThread:3912 [handler.py:handle_request():146] handle_request: stop_status
2025-03-09 22:17:51,648 DEBUG   SenderThread:3912 [sender.py:send_request():409] send_request: stop_status
2025-03-09 22:17:51,954 DEBUG   HandlerThread:3912 [handler.py:handle_request():146] handle_request: status_report
2025-03-09 22:17:53,139 ERROR   gpu       :3912 [interfaces.py:monitor():144] Failed to sample metric: Unknown Error
2025-03-09 22:17:53,140 ERROR   gpu       :3912 [interfaces.py:monitor():144] Failed to sample metric: Unknown Error
2025-03-09 22:17:55,565 ERROR   gpu       :3912 [interfaces.py:monitor():144] Failed to sample metric: Unknown Error
2025-03-09 22:17:55,566 ERROR   gpu       :3912 [interfaces.py:monitor():144] Failed to sample metric: Unknown Error
2025-03-09 22:17:57,014 DEBUG   HandlerThread:3912 [handler.py:handle_request():146] handle_request: status_report
2025-03-09 22:17:58,031 ERROR   gpu       :3912 [interfaces.py:monitor():144] Failed to sample metric: Unknown Error
2025-03-09 22:17:58,032 ERROR   gpu       :3912 [interfaces.py:monitor():144] Failed to sample metric: Unknown Error
2025-03-09 22:18:00,464 ERROR   gpu       :3912 [interfaces.py:monitor():144] Failed to sample metric: Unknown Error
2025-03-09 22:18:00,465 ERROR   gpu       :3912 [interfaces.py:monitor():144] Failed to sample metric: Unknown Error
2025-03-09 22:18:02,079 DEBUG   HandlerThread:3912 [handler.py:handle_request():146] handle_request: status_report
2025-03-09 22:18:02,890 ERROR   gpu       :3912 [interfaces.py:monitor():144] Failed to sample metric: Unknown Error
2025-03-09 22:18:02,891 ERROR   gpu       :3912 [interfaces.py:monitor():144] Failed to sample metric: Unknown Error
2025-03-09 22:18:03,105 INFO    Thread-16 :3912 [dir_watcher.py:_on_file_modified():288] file/dir modified: c:\Users\D. Sai Mani Kumar\Downloads\DL\da6401_assignment1\wandb\run-20250309_221730-el3kt5nf\files\config.yaml
2025-03-09 22:18:05,331 ERROR   gpu       :3912 [interfaces.py:monitor():144] Failed to sample metric: Unknown Error
2025-03-09 22:18:05,332 ERROR   gpu       :3912 [interfaces.py:monitor():144] Failed to sample metric: Unknown Error
2025-03-09 22:18:06,662 DEBUG   HandlerThread:3912 [handler.py:handle_request():146] handle_request: stop_status
2025-03-09 22:18:06,662 DEBUG   HandlerThread:3912 [handler.py:handle_request():146] handle_request: internal_messages
2025-03-09 22:18:06,664 DEBUG   SenderThread:3912 [sender.py:send_request():409] send_request: stop_status
2025-03-09 22:18:07,768 ERROR   gpu       :3912 [interfaces.py:monitor():144] Failed to sample metric: Unknown Error
2025-03-09 22:18:07,769 ERROR   gpu       :3912 [interfaces.py:monitor():144] Failed to sample metric: Unknown Error
2025-03-09 22:18:07,989 DEBUG   HandlerThread:3912 [handler.py:handle_request():146] handle_request: status_report
2025-03-09 22:18:10,190 ERROR   gpu       :3912 [interfaces.py:monitor():144] Failed to sample metric: Unknown Error
2025-03-09 22:18:10,191 ERROR   gpu       :3912 [interfaces.py:monitor():144] Failed to sample metric: Unknown Error
2025-03-09 22:18:13,039 DEBUG   HandlerThread:3912 [handler.py:handle_request():146] handle_request: status_report
2025-03-09 22:18:15,187 ERROR   gpu       :3912 [interfaces.py:monitor():144] Failed to sample metric: Unknown Error
2025-03-09 22:18:15,188 ERROR   gpu       :3912 [interfaces.py:monitor():144] Failed to sample metric: Unknown Error
2025-03-09 22:18:17,696 ERROR   gpu       :3912 [interfaces.py:monitor():144] Failed to sample metric: Unknown Error
2025-03-09 22:18:17,697 ERROR   gpu       :3912 [interfaces.py:monitor():144] Failed to sample metric: Unknown Error
2025-03-09 22:18:17,884 DEBUG   HandlerThread:3912 [handler.py:handle_request():146] handle_request: partial_history
2025-03-09 22:18:17,886 DEBUG   SenderThread:3912 [sender.py:send():382] send: history
2025-03-09 22:18:17,886 DEBUG   SenderThread:3912 [sender.py:send_request():409] send_request: summary_record
2025-03-09 22:18:17,889 INFO    SenderThread:3912 [sender.py:_save_file():1392] saving file wandb-summary.json with policy end
2025-03-09 22:18:18,576 INFO    Thread-16 :3912 [dir_watcher.py:_on_file_created():271] file/dir created: c:\Users\D. Sai Mani Kumar\Downloads\DL\da6401_assignment1\wandb\run-20250309_221730-el3kt5nf\files\wandb-summary.json
2025-03-09 22:18:18,895 DEBUG   HandlerThread:3912 [handler.py:handle_request():146] handle_request: status_report
2025-03-09 22:18:20,132 ERROR   gpu       :3912 [interfaces.py:monitor():144] Failed to sample metric: Unknown Error
2025-03-09 22:18:20,133 ERROR   gpu       :3912 [interfaces.py:monitor():144] Failed to sample metric: Unknown Error
2025-03-09 22:18:20,604 INFO    Thread-16 :3912 [dir_watcher.py:_on_file_modified():288] file/dir modified: c:\Users\D. Sai Mani Kumar\Downloads\DL\da6401_assignment1\wandb\run-20250309_221730-el3kt5nf\files\output.log
2025-03-09 22:18:21,679 DEBUG   HandlerThread:3912 [handler.py:handle_request():146] handle_request: stop_status
2025-03-09 22:18:21,680 DEBUG   HandlerThread:3912 [handler.py:handle_request():146] handle_request: internal_messages
2025-03-09 22:18:21,681 DEBUG   SenderThread:3912 [sender.py:send_request():409] send_request: stop_status
2025-03-09 22:18:22,551 ERROR   gpu       :3912 [interfaces.py:monitor():144] Failed to sample metric: Unknown Error
2025-03-09 22:18:22,552 ERROR   gpu       :3912 [interfaces.py:monitor():144] Failed to sample metric: Unknown Error
2025-03-09 22:18:24,117 DEBUG   HandlerThread:3912 [handler.py:handle_request():146] handle_request: status_report
2025-03-09 22:18:24,989 ERROR   gpu       :3912 [interfaces.py:monitor():144] Failed to sample metric: Unknown Error
2025-03-09 22:18:24,991 ERROR   gpu       :3912 [interfaces.py:monitor():144] Failed to sample metric: Unknown Error
2025-03-09 22:18:27,422 ERROR   gpu       :3912 [interfaces.py:monitor():144] Failed to sample metric: Unknown Error
2025-03-09 22:18:27,424 ERROR   gpu       :3912 [interfaces.py:monitor():144] Failed to sample metric: Unknown Error
2025-03-09 22:18:29,162 DEBUG   HandlerThread:3912 [handler.py:handle_request():146] handle_request: status_report
2025-03-09 22:18:29,845 ERROR   gpu       :3912 [interfaces.py:monitor():144] Failed to sample metric: Unknown Error
2025-03-09 22:18:29,846 ERROR   gpu       :3912 [interfaces.py:monitor():144] Failed to sample metric: Unknown Error
2025-03-09 22:18:32,272 ERROR   gpu       :3912 [interfaces.py:monitor():144] Failed to sample metric: Unknown Error
2025-03-09 22:18:32,272 ERROR   gpu       :3912 [interfaces.py:monitor():144] Failed to sample metric: Unknown Error
2025-03-09 22:18:34,206 DEBUG   HandlerThread:3912 [handler.py:handle_request():146] handle_request: status_report
2025-03-09 22:18:34,694 ERROR   gpu       :3912 [interfaces.py:monitor():144] Failed to sample metric: Unknown Error
2025-03-09 22:18:34,695 ERROR   gpu       :3912 [interfaces.py:monitor():144] Failed to sample metric: Unknown Error
2025-03-09 22:18:36,244 DEBUG   SystemMonitor:3912 [system_monitor.py:_start():172] Starting system metrics aggregation loop
2025-03-09 22:18:36,246 DEBUG   SenderThread:3912 [sender.py:send():382] send: stats
2025-03-09 22:18:36,682 DEBUG   HandlerThread:3912 [handler.py:handle_request():146] handle_request: internal_messages
2025-03-09 22:18:36,697 DEBUG   HandlerThread:3912 [handler.py:handle_request():146] handle_request: stop_status
2025-03-09 22:18:36,698 DEBUG   SenderThread:3912 [sender.py:send_request():409] send_request: stop_status
2025-03-09 22:18:37,119 ERROR   gpu       :3912 [interfaces.py:monitor():144] Failed to sample metric: Unknown Error
2025-03-09 22:18:37,119 ERROR   gpu       :3912 [interfaces.py:monitor():144] Failed to sample metric: Unknown Error
2025-03-09 22:18:39,560 ERROR   gpu       :3912 [interfaces.py:monitor():144] Failed to sample metric: Unknown Error
2025-03-09 22:18:39,562 ERROR   gpu       :3912 [interfaces.py:monitor():144] Failed to sample metric: Unknown Error
2025-03-09 22:18:40,079 DEBUG   HandlerThread:3912 [handler.py:handle_request():146] handle_request: status_report
2025-03-09 22:18:42,050 ERROR   gpu       :3912 [interfaces.py:monitor():144] Failed to sample metric: Unknown Error
2025-03-09 22:18:42,050 ERROR   gpu       :3912 [interfaces.py:monitor():144] Failed to sample metric: Unknown Error
2025-03-09 22:18:44,498 ERROR   gpu       :3912 [interfaces.py:monitor():144] Failed to sample metric: Unknown Error
2025-03-09 22:18:44,500 ERROR   gpu       :3912 [interfaces.py:monitor():144] Failed to sample metric: Unknown Error
2025-03-09 22:18:45,125 DEBUG   HandlerThread:3912 [handler.py:handle_request():146] handle_request: status_report
2025-03-09 22:18:46,976 ERROR   gpu       :3912 [interfaces.py:monitor():144] Failed to sample metric: Unknown Error
2025-03-09 22:18:46,976 ERROR   gpu       :3912 [interfaces.py:monitor():144] Failed to sample metric: Unknown Error
2025-03-09 22:18:50,180 DEBUG   HandlerThread:3912 [handler.py:handle_request():146] handle_request: status_report
2025-03-09 22:18:51,680 DEBUG   HandlerThread:3912 [handler.py:handle_request():146] handle_request: internal_messages
2025-03-09 22:18:51,726 DEBUG   HandlerThread:3912 [handler.py:handle_request():146] handle_request: stop_status
2025-03-09 22:18:51,728 DEBUG   SenderThread:3912 [sender.py:send_request():409] send_request: stop_status
2025-03-09 22:18:51,959 ERROR   gpu       :3912 [interfaces.py:monitor():144] Failed to sample metric: Unknown Error
2025-03-09 22:18:51,959 ERROR   gpu       :3912 [interfaces.py:monitor():144] Failed to sample metric: Unknown Error
2025-03-09 22:18:54,392 ERROR   gpu       :3912 [interfaces.py:monitor():144] Failed to sample metric: Unknown Error
2025-03-09 22:18:54,393 ERROR   gpu       :3912 [interfaces.py:monitor():144] Failed to sample metric: Unknown Error
2025-03-09 22:18:54,537 DEBUG   HandlerThread:3912 [handler.py:handle_request():146] handle_request: partial_history
2025-03-09 22:18:54,538 DEBUG   SenderThread:3912 [sender.py:send():382] send: history
2025-03-09 22:18:54,540 DEBUG   SenderThread:3912 [sender.py:send_request():409] send_request: summary_record
2025-03-09 22:18:54,547 INFO    SenderThread:3912 [sender.py:_save_file():1392] saving file wandb-summary.json with policy end
2025-03-09 22:18:55,066 INFO    Thread-16 :3912 [dir_watcher.py:_on_file_modified():288] file/dir modified: c:\Users\D. Sai Mani Kumar\Downloads\DL\da6401_assignment1\wandb\run-20250309_221730-el3kt5nf\files\wandb-summary.json
2025-03-09 22:18:55,565 DEBUG   HandlerThread:3912 [handler.py:handle_request():146] handle_request: status_report
2025-03-09 22:18:56,085 INFO    Thread-16 :3912 [dir_watcher.py:_on_file_modified():288] file/dir modified: c:\Users\D. Sai Mani Kumar\Downloads\DL\da6401_assignment1\wandb\run-20250309_221730-el3kt5nf\files\output.log
2025-03-09 22:18:56,843 ERROR   gpu       :3912 [interfaces.py:monitor():144] Failed to sample metric: Unknown Error
2025-03-09 22:18:56,843 ERROR   gpu       :3912 [interfaces.py:monitor():144] Failed to sample metric: Unknown Error
2025-03-09 22:18:59,264 ERROR   gpu       :3912 [interfaces.py:monitor():144] Failed to sample metric: Unknown Error
2025-03-09 22:18:59,265 ERROR   gpu       :3912 [interfaces.py:monitor():144] Failed to sample metric: Unknown Error
2025-03-09 22:19:00,611 DEBUG   HandlerThread:3912 [handler.py:handle_request():146] handle_request: status_report
2025-03-09 22:19:01,692 ERROR   gpu       :3912 [interfaces.py:monitor():144] Failed to sample metric: Unknown Error
2025-03-09 22:19:01,693 ERROR   gpu       :3912 [interfaces.py:monitor():144] Failed to sample metric: Unknown Error
2025-03-09 22:19:04,116 ERROR   gpu       :3912 [interfaces.py:monitor():144] Failed to sample metric: Unknown Error
2025-03-09 22:19:04,117 ERROR   gpu       :3912 [interfaces.py:monitor():144] Failed to sample metric: Unknown Error
2025-03-09 22:19:05,638 DEBUG   HandlerThread:3912 [handler.py:handle_request():146] handle_request: status_report
2025-03-09 22:19:06,257 DEBUG   SenderThread:3912 [sender.py:send():382] send: stats
2025-03-09 22:19:06,536 ERROR   gpu       :3912 [interfaces.py:monitor():144] Failed to sample metric: Unknown Error
2025-03-09 22:19:06,537 ERROR   gpu       :3912 [interfaces.py:monitor():144] Failed to sample metric: Unknown Error
2025-03-09 22:19:06,695 DEBUG   HandlerThread:3912 [handler.py:handle_request():146] handle_request: internal_messages
2025-03-09 22:19:06,710 DEBUG   HandlerThread:3912 [handler.py:handle_request():146] handle_request: stop_status
2025-03-09 22:19:06,711 DEBUG   SenderThread:3912 [sender.py:send_request():409] send_request: stop_status
2025-03-09 22:19:08,963 ERROR   gpu       :3912 [interfaces.py:monitor():144] Failed to sample metric: Unknown Error
2025-03-09 22:19:08,964 ERROR   gpu       :3912 [interfaces.py:monitor():144] Failed to sample metric: Unknown Error
2025-03-09 22:19:11,145 DEBUG   HandlerThread:3912 [handler.py:handle_request():146] handle_request: status_report
2025-03-09 22:19:11,379 ERROR   gpu       :3912 [interfaces.py:monitor():144] Failed to sample metric: Unknown Error
2025-03-09 22:19:11,379 ERROR   gpu       :3912 [interfaces.py:monitor():144] Failed to sample metric: Unknown Error
2025-03-09 22:19:13,813 ERROR   gpu       :3912 [interfaces.py:monitor():144] Failed to sample metric: Unknown Error
2025-03-09 22:19:13,814 ERROR   gpu       :3912 [interfaces.py:monitor():144] Failed to sample metric: Unknown Error
2025-03-09 22:19:16,214 DEBUG   HandlerThread:3912 [handler.py:handle_request():146] handle_request: status_report
2025-03-09 22:19:16,229 ERROR   gpu       :3912 [interfaces.py:monitor():144] Failed to sample metric: Unknown Error
2025-03-09 22:19:16,231 ERROR   gpu       :3912 [interfaces.py:monitor():144] Failed to sample metric: Unknown Error
2025-03-09 22:19:18,684 ERROR   gpu       :3912 [interfaces.py:monitor():144] Failed to sample metric: Unknown Error
2025-03-09 22:19:18,687 ERROR   gpu       :3912 [interfaces.py:monitor():144] Failed to sample metric: Unknown Error
2025-03-09 22:19:21,132 ERROR   gpu       :3912 [interfaces.py:monitor():144] Failed to sample metric: Unknown Error
2025-03-09 22:19:21,133 ERROR   gpu       :3912 [interfaces.py:monitor():144] Failed to sample metric: Unknown Error
2025-03-09 22:19:21,290 DEBUG   HandlerThread:3912 [handler.py:handle_request():146] handle_request: status_report
2025-03-09 22:19:21,712 DEBUG   HandlerThread:3912 [handler.py:handle_request():146] handle_request: internal_messages
2025-03-09 22:19:21,714 DEBUG   HandlerThread:3912 [handler.py:handle_request():146] handle_request: stop_status
2025-03-09 22:19:21,716 DEBUG   SenderThread:3912 [sender.py:send_request():409] send_request: stop_status
2025-03-09 22:19:23,577 ERROR   gpu       :3912 [interfaces.py:monitor():144] Failed to sample metric: Unknown Error
2025-03-09 22:19:23,578 ERROR   gpu       :3912 [interfaces.py:monitor():144] Failed to sample metric: Unknown Error
2025-03-09 22:19:27,130 DEBUG   HandlerThread:3912 [handler.py:handle_request():146] handle_request: status_report
2025-03-09 22:19:28,552 ERROR   gpu       :3912 [interfaces.py:monitor():144] Failed to sample metric: Unknown Error
2025-03-09 22:19:28,553 ERROR   gpu       :3912 [interfaces.py:monitor():144] Failed to sample metric: Unknown Error
2025-03-09 22:19:30,985 ERROR   gpu       :3912 [interfaces.py:monitor():144] Failed to sample metric: Unknown Error
2025-03-09 22:19:30,987 ERROR   gpu       :3912 [interfaces.py:monitor():144] Failed to sample metric: Unknown Error
2025-03-09 22:19:32,204 DEBUG   HandlerThread:3912 [handler.py:handle_request():146] handle_request: status_report
2025-03-09 22:19:33,426 ERROR   gpu       :3912 [interfaces.py:monitor():144] Failed to sample metric: Unknown Error
2025-03-09 22:19:33,426 ERROR   gpu       :3912 [interfaces.py:monitor():144] Failed to sample metric: Unknown Error
2025-03-09 22:19:35,862 ERROR   gpu       :3912 [interfaces.py:monitor():144] Failed to sample metric: Unknown Error
2025-03-09 22:19:35,863 ERROR   gpu       :3912 [interfaces.py:monitor():144] Failed to sample metric: Unknown Error
2025-03-09 22:19:36,269 DEBUG   SenderThread:3912 [sender.py:send():382] send: stats
2025-03-09 22:19:36,722 DEBUG   HandlerThread:3912 [handler.py:handle_request():146] handle_request: stop_status
2025-03-09 22:19:36,722 DEBUG   HandlerThread:3912 [handler.py:handle_request():146] handle_request: internal_messages
2025-03-09 22:19:36,723 DEBUG   SenderThread:3912 [sender.py:send_request():409] send_request: stop_status
2025-03-09 22:19:38,112 DEBUG   HandlerThread:3912 [handler.py:handle_request():146] handle_request: status_report
2025-03-09 22:19:38,284 ERROR   gpu       :3912 [interfaces.py:monitor():144] Failed to sample metric: Unknown Error
2025-03-09 22:19:38,284 ERROR   gpu       :3912 [interfaces.py:monitor():144] Failed to sample metric: Unknown Error
2025-03-09 22:19:40,721 ERROR   gpu       :3912 [interfaces.py:monitor():144] Failed to sample metric: Unknown Error
2025-03-09 22:19:40,722 ERROR   gpu       :3912 [interfaces.py:monitor():144] Failed to sample metric: Unknown Error
2025-03-09 22:19:43,144 DEBUG   HandlerThread:3912 [handler.py:handle_request():146] handle_request: status_report
2025-03-09 22:19:43,145 ERROR   gpu       :3912 [interfaces.py:monitor():144] Failed to sample metric: Unknown Error
2025-03-09 22:19:43,146 ERROR   gpu       :3912 [interfaces.py:monitor():144] Failed to sample metric: Unknown Error
2025-03-09 22:19:45,576 ERROR   gpu       :3912 [interfaces.py:monitor():144] Failed to sample metric: Unknown Error
2025-03-09 22:19:45,577 ERROR   gpu       :3912 [interfaces.py:monitor():144] Failed to sample metric: Unknown Error
2025-03-09 22:19:48,007 ERROR   gpu       :3912 [interfaces.py:monitor():144] Failed to sample metric: Unknown Error
2025-03-09 22:19:48,007 ERROR   gpu       :3912 [interfaces.py:monitor():144] Failed to sample metric: Unknown Error
2025-03-09 22:19:48,179 DEBUG   HandlerThread:3912 [handler.py:handle_request():146] handle_request: status_report
2025-03-09 22:19:50,433 ERROR   gpu       :3912 [interfaces.py:monitor():144] Failed to sample metric: Unknown Error
2025-03-09 22:19:50,434 ERROR   gpu       :3912 [interfaces.py:monitor():144] Failed to sample metric: Unknown Error
2025-03-09 22:19:51,737 DEBUG   HandlerThread:3912 [handler.py:handle_request():146] handle_request: internal_messages
2025-03-09 22:19:51,737 DEBUG   HandlerThread:3912 [handler.py:handle_request():146] handle_request: stop_status
2025-03-09 22:19:51,739 DEBUG   SenderThread:3912 [sender.py:send_request():409] send_request: stop_status
2025-03-09 22:19:52,845 ERROR   gpu       :3912 [interfaces.py:monitor():144] Failed to sample metric: Unknown Error
2025-03-09 22:19:52,846 ERROR   gpu       :3912 [interfaces.py:monitor():144] Failed to sample metric: Unknown Error
2025-03-09 22:19:54,139 DEBUG   HandlerThread:3912 [handler.py:handle_request():146] handle_request: status_report
2025-03-09 22:19:55,279 ERROR   gpu       :3912 [interfaces.py:monitor():144] Failed to sample metric: Unknown Error
2025-03-09 22:19:55,280 ERROR   gpu       :3912 [interfaces.py:monitor():144] Failed to sample metric: Unknown Error
2025-03-09 22:19:57,728 ERROR   gpu       :3912 [interfaces.py:monitor():144] Failed to sample metric: Unknown Error
2025-03-09 22:19:57,729 ERROR   gpu       :3912 [interfaces.py:monitor():144] Failed to sample metric: Unknown Error
2025-03-09 22:19:59,227 DEBUG   HandlerThread:3912 [handler.py:handle_request():146] handle_request: status_report
2025-03-09 22:20:00,165 ERROR   gpu       :3912 [interfaces.py:monitor():144] Failed to sample metric: Unknown Error
2025-03-09 22:20:00,166 ERROR   gpu       :3912 [interfaces.py:monitor():144] Failed to sample metric: Unknown Error
2025-03-09 22:20:04,272 DEBUG   HandlerThread:3912 [handler.py:handle_request():146] handle_request: status_report
2025-03-09 22:20:05,147 ERROR   gpu       :3912 [interfaces.py:monitor():144] Failed to sample metric: Unknown Error
2025-03-09 22:20:05,147 ERROR   gpu       :3912 [interfaces.py:monitor():144] Failed to sample metric: Unknown Error
2025-03-09 22:20:05,888 DEBUG   HandlerThread:3912 [handler.py:handle_request():146] handle_request: partial_history
2025-03-09 22:20:05,889 DEBUG   SenderThread:3912 [sender.py:send():382] send: history
2025-03-09 22:20:05,890 DEBUG   SenderThread:3912 [sender.py:send_request():409] send_request: summary_record
2025-03-09 22:20:05,894 INFO    SenderThread:3912 [sender.py:_save_file():1392] saving file wandb-summary.json with policy end
2025-03-09 22:20:06,037 INFO    Thread-16 :3912 [dir_watcher.py:_on_file_modified():288] file/dir modified: c:\Users\D. Sai Mani Kumar\Downloads\DL\da6401_assignment1\wandb\run-20250309_221730-el3kt5nf\files\wandb-summary.json
2025-03-09 22:20:06,272 DEBUG   SenderThread:3912 [sender.py:send():382] send: stats
2025-03-09 22:20:06,742 DEBUG   HandlerThread:3912 [handler.py:handle_request():146] handle_request: stop_status
2025-03-09 22:20:06,742 DEBUG   HandlerThread:3912 [handler.py:handle_request():146] handle_request: internal_messages
2025-03-09 22:20:06,743 DEBUG   SenderThread:3912 [sender.py:send_request():409] send_request: stop_status
2025-03-09 22:20:07,072 INFO    Thread-16 :3912 [dir_watcher.py:_on_file_modified():288] file/dir modified: c:\Users\D. Sai Mani Kumar\Downloads\DL\da6401_assignment1\wandb\run-20250309_221730-el3kt5nf\files\output.log
2025-03-09 22:20:07,567 ERROR   gpu       :3912 [interfaces.py:monitor():144] Failed to sample metric: Unknown Error
2025-03-09 22:20:07,568 ERROR   gpu       :3912 [interfaces.py:monitor():144] Failed to sample metric: Unknown Error
2025-03-09 22:20:09,394 DEBUG   HandlerThread:3912 [handler.py:handle_request():146] handle_request: status_report
2025-03-09 22:20:09,991 ERROR   gpu       :3912 [interfaces.py:monitor():144] Failed to sample metric: Unknown Error
2025-03-09 22:20:09,992 ERROR   gpu       :3912 [interfaces.py:monitor():144] Failed to sample metric: Unknown Error
2025-03-09 22:20:12,421 ERROR   gpu       :3912 [interfaces.py:monitor():144] Failed to sample metric: Unknown Error
2025-03-09 22:20:12,422 ERROR   gpu       :3912 [interfaces.py:monitor():144] Failed to sample metric: Unknown Error
2025-03-09 22:20:14,448 DEBUG   HandlerThread:3912 [handler.py:handle_request():146] handle_request: status_report
2025-03-09 22:20:14,838 ERROR   gpu       :3912 [interfaces.py:monitor():144] Failed to sample metric: Unknown Error
2025-03-09 22:20:14,839 ERROR   gpu       :3912 [interfaces.py:monitor():144] Failed to sample metric: Unknown Error
2025-03-09 22:20:17,289 ERROR   gpu       :3912 [interfaces.py:monitor():144] Failed to sample metric: Unknown Error
2025-03-09 22:20:17,290 ERROR   gpu       :3912 [interfaces.py:monitor():144] Failed to sample metric: Unknown Error
2025-03-09 22:20:19,474 DEBUG   HandlerThread:3912 [handler.py:handle_request():146] handle_request: status_report
2025-03-09 22:20:19,704 ERROR   gpu       :3912 [interfaces.py:monitor():144] Failed to sample metric: Unknown Error
2025-03-09 22:20:19,705 ERROR   gpu       :3912 [interfaces.py:monitor():144] Failed to sample metric: Unknown Error
2025-03-09 22:20:21,741 DEBUG   HandlerThread:3912 [handler.py:handle_request():146] handle_request: stop_status
2025-03-09 22:20:21,742 DEBUG   HandlerThread:3912 [handler.py:handle_request():146] handle_request: internal_messages
2025-03-09 22:20:21,743 DEBUG   SenderThread:3912 [sender.py:send_request():409] send_request: stop_status
2025-03-09 22:20:22,126 ERROR   gpu       :3912 [interfaces.py:monitor():144] Failed to sample metric: Unknown Error
2025-03-09 22:20:22,127 ERROR   gpu       :3912 [interfaces.py:monitor():144] Failed to sample metric: Unknown Error
2025-03-09 22:20:24,554 ERROR   gpu       :3912 [interfaces.py:monitor():144] Failed to sample metric: Unknown Error
2025-03-09 22:20:24,555 ERROR   gpu       :3912 [interfaces.py:monitor():144] Failed to sample metric: Unknown Error
2025-03-09 22:20:25,097 DEBUG   HandlerThread:3912 [handler.py:handle_request():146] handle_request: status_report
2025-03-09 22:20:26,982 ERROR   gpu       :3912 [interfaces.py:monitor():144] Failed to sample metric: Unknown Error
2025-03-09 22:20:26,982 ERROR   gpu       :3912 [interfaces.py:monitor():144] Failed to sample metric: Unknown Error
2025-03-09 22:20:29,595 ERROR   gpu       :3912 [interfaces.py:monitor():144] Failed to sample metric: Unknown Error
2025-03-09 22:20:29,596 ERROR   gpu       :3912 [interfaces.py:monitor():144] Failed to sample metric: Unknown Error
2025-03-09 22:20:30,122 DEBUG   HandlerThread:3912 [handler.py:handle_request():146] handle_request: status_report
2025-03-09 22:20:32,028 ERROR   gpu       :3912 [interfaces.py:monitor():144] Failed to sample metric: Unknown Error
2025-03-09 22:20:32,029 ERROR   gpu       :3912 [interfaces.py:monitor():144] Failed to sample metric: Unknown Error
2025-03-09 22:20:34,572 ERROR   gpu       :3912 [interfaces.py:monitor():144] Failed to sample metric: Unknown Error
2025-03-09 22:20:34,573 ERROR   gpu       :3912 [interfaces.py:monitor():144] Failed to sample metric: Unknown Error
2025-03-09 22:20:35,183 DEBUG   HandlerThread:3912 [handler.py:handle_request():146] handle_request: status_report
2025-03-09 22:20:36,278 DEBUG   SenderThread:3912 [sender.py:send():382] send: stats
2025-03-09 22:20:36,761 DEBUG   HandlerThread:3912 [handler.py:handle_request():146] handle_request: stop_status
2025-03-09 22:20:36,762 DEBUG   HandlerThread:3912 [handler.py:handle_request():146] handle_request: internal_messages
2025-03-09 22:20:36,763 DEBUG   SenderThread:3912 [sender.py:send_request():409] send_request: stop_status
2025-03-09 22:20:37,011 ERROR   gpu       :3912 [interfaces.py:monitor():144] Failed to sample metric: Unknown Error
2025-03-09 22:20:37,012 ERROR   gpu       :3912 [interfaces.py:monitor():144] Failed to sample metric: Unknown Error
2025-03-09 22:20:41,173 DEBUG   HandlerThread:3912 [handler.py:handle_request():146] handle_request: status_report
2025-03-09 22:20:42,034 ERROR   gpu       :3912 [interfaces.py:monitor():144] Failed to sample metric: Unknown Error
2025-03-09 22:20:42,035 ERROR   gpu       :3912 [interfaces.py:monitor():144] Failed to sample metric: Unknown Error
2025-03-09 22:20:44,449 ERROR   gpu       :3912 [interfaces.py:monitor():144] Failed to sample metric: Unknown Error
2025-03-09 22:20:44,450 ERROR   gpu       :3912 [interfaces.py:monitor():144] Failed to sample metric: Unknown Error
2025-03-09 22:20:46,195 DEBUG   HandlerThread:3912 [handler.py:handle_request():146] handle_request: status_report
2025-03-09 22:20:46,867 ERROR   gpu       :3912 [interfaces.py:monitor():144] Failed to sample metric: Unknown Error
2025-03-09 22:20:46,868 ERROR   gpu       :3912 [interfaces.py:monitor():144] Failed to sample metric: Unknown Error
2025-03-09 22:20:49,296 ERROR   gpu       :3912 [interfaces.py:monitor():144] Failed to sample metric: Unknown Error
2025-03-09 22:20:49,296 ERROR   gpu       :3912 [interfaces.py:monitor():144] Failed to sample metric: Unknown Error
2025-03-09 22:20:51,259 DEBUG   HandlerThread:3912 [handler.py:handle_request():146] handle_request: status_report
2025-03-09 22:20:51,711 ERROR   gpu       :3912 [interfaces.py:monitor():144] Failed to sample metric: Unknown Error
2025-03-09 22:20:51,712 ERROR   gpu       :3912 [interfaces.py:monitor():144] Failed to sample metric: Unknown Error
2025-03-09 22:20:51,757 DEBUG   HandlerThread:3912 [handler.py:handle_request():146] handle_request: internal_messages
2025-03-09 22:20:51,758 DEBUG   HandlerThread:3912 [handler.py:handle_request():146] handle_request: stop_status
2025-03-09 22:20:51,759 DEBUG   SenderThread:3912 [sender.py:send_request():409] send_request: stop_status
2025-03-09 22:20:54,158 ERROR   gpu       :3912 [interfaces.py:monitor():144] Failed to sample metric: Unknown Error
2025-03-09 22:20:54,160 ERROR   gpu       :3912 [interfaces.py:monitor():144] Failed to sample metric: Unknown Error
2025-03-09 22:20:56,581 ERROR   gpu       :3912 [interfaces.py:monitor():144] Failed to sample metric: Unknown Error
2025-03-09 22:20:56,582 ERROR   gpu       :3912 [interfaces.py:monitor():144] Failed to sample metric: Unknown Error
2025-03-09 22:20:57,217 DEBUG   HandlerThread:3912 [handler.py:handle_request():146] handle_request: status_report
2025-03-09 22:20:58,999 ERROR   gpu       :3912 [interfaces.py:monitor():144] Failed to sample metric: Unknown Error
2025-03-09 22:20:59,000 ERROR   gpu       :3912 [interfaces.py:monitor():144] Failed to sample metric: Unknown Error
2025-03-09 22:21:01,419 ERROR   gpu       :3912 [interfaces.py:monitor():144] Failed to sample metric: Unknown Error
2025-03-09 22:21:01,421 ERROR   gpu       :3912 [interfaces.py:monitor():144] Failed to sample metric: Unknown Error
2025-03-09 22:21:02,276 DEBUG   HandlerThread:3912 [handler.py:handle_request():146] handle_request: status_report
2025-03-09 22:21:03,875 ERROR   gpu       :3912 [interfaces.py:monitor():144] Failed to sample metric: Unknown Error
2025-03-09 22:21:03,875 ERROR   gpu       :3912 [interfaces.py:monitor():144] Failed to sample metric: Unknown Error
2025-03-09 22:21:06,291 DEBUG   SenderThread:3912 [sender.py:send():382] send: stats
2025-03-09 22:21:06,306 ERROR   gpu       :3912 [interfaces.py:monitor():144] Failed to sample metric: Unknown Error
2025-03-09 22:21:06,308 ERROR   gpu       :3912 [interfaces.py:monitor():144] Failed to sample metric: Unknown Error
2025-03-09 22:21:06,758 DEBUG   HandlerThread:3912 [handler.py:handle_request():146] handle_request: stop_status
2025-03-09 22:21:06,760 DEBUG   SenderThread:3912 [sender.py:send_request():409] send_request: stop_status
2025-03-09 22:21:06,774 DEBUG   HandlerThread:3912 [handler.py:handle_request():146] handle_request: internal_messages
2025-03-09 22:21:08,089 DEBUG   HandlerThread:3912 [handler.py:handle_request():146] handle_request: status_report
2025-03-09 22:21:08,752 ERROR   gpu       :3912 [interfaces.py:monitor():144] Failed to sample metric: Unknown Error
2025-03-09 22:21:08,753 ERROR   gpu       :3912 [interfaces.py:monitor():144] Failed to sample metric: Unknown Error
2025-03-09 22:21:11,181 ERROR   gpu       :3912 [interfaces.py:monitor():144] Failed to sample metric: Unknown Error
2025-03-09 22:21:11,182 ERROR   gpu       :3912 [interfaces.py:monitor():144] Failed to sample metric: Unknown Error
2025-03-09 22:21:13,133 DEBUG   HandlerThread:3912 [handler.py:handle_request():146] handle_request: status_report
2025-03-09 22:21:13,600 ERROR   gpu       :3912 [interfaces.py:monitor():144] Failed to sample metric: Unknown Error
2025-03-09 22:21:13,600 ERROR   gpu       :3912 [interfaces.py:monitor():144] Failed to sample metric: Unknown Error
2025-03-09 22:21:13,930 DEBUG   HandlerThread:3912 [handler.py:handle_request():146] handle_request: partial_history
2025-03-09 22:21:13,932 DEBUG   SenderThread:3912 [sender.py:send():382] send: history
2025-03-09 22:21:13,932 DEBUG   SenderThread:3912 [sender.py:send_request():409] send_request: summary_record
2025-03-09 22:21:13,934 INFO    SenderThread:3912 [sender.py:_save_file():1392] saving file wandb-summary.json with policy end
2025-03-09 22:21:14,931 INFO    Thread-16 :3912 [dir_watcher.py:_on_file_modified():288] file/dir modified: c:\Users\D. Sai Mani Kumar\Downloads\DL\da6401_assignment1\wandb\run-20250309_221730-el3kt5nf\files\wandb-summary.json
2025-03-09 22:21:14,931 INFO    Thread-16 :3912 [dir_watcher.py:_on_file_modified():288] file/dir modified: c:\Users\D. Sai Mani Kumar\Downloads\DL\da6401_assignment1\wandb\run-20250309_221730-el3kt5nf\files\output.log
2025-03-09 22:21:18,569 ERROR   gpu       :3912 [interfaces.py:monitor():144] Failed to sample metric: Unknown Error
2025-03-09 22:21:18,569 ERROR   gpu       :3912 [interfaces.py:monitor():144] Failed to sample metric: Unknown Error
2025-03-09 22:21:19,038 DEBUG   HandlerThread:3912 [handler.py:handle_request():146] handle_request: status_report
2025-03-09 22:21:21,114 ERROR   gpu       :3912 [interfaces.py:monitor():144] Failed to sample metric: Unknown Error
2025-03-09 22:21:21,116 ERROR   gpu       :3912 [interfaces.py:monitor():144] Failed to sample metric: Unknown Error
2025-03-09 22:21:21,773 DEBUG   HandlerThread:3912 [handler.py:handle_request():146] handle_request: stop_status
2025-03-09 22:21:21,775 DEBUG   SenderThread:3912 [sender.py:send_request():409] send_request: stop_status
2025-03-09 22:21:21,788 DEBUG   HandlerThread:3912 [handler.py:handle_request():146] handle_request: internal_messages
2025-03-09 22:21:23,550 ERROR   gpu       :3912 [interfaces.py:monitor():144] Failed to sample metric: Unknown Error
2025-03-09 22:21:23,550 ERROR   gpu       :3912 [interfaces.py:monitor():144] Failed to sample metric: Unknown Error
2025-03-09 22:21:24,139 DEBUG   HandlerThread:3912 [handler.py:handle_request():146] handle_request: status_report
2025-03-09 22:21:25,967 ERROR   gpu       :3912 [interfaces.py:monitor():144] Failed to sample metric: Unknown Error
2025-03-09 22:21:25,967 ERROR   gpu       :3912 [interfaces.py:monitor():144] Failed to sample metric: Unknown Error
2025-03-09 22:21:28,394 ERROR   gpu       :3912 [interfaces.py:monitor():144] Failed to sample metric: Unknown Error
2025-03-09 22:21:28,394 ERROR   gpu       :3912 [interfaces.py:monitor():144] Failed to sample metric: Unknown Error
2025-03-09 22:21:29,222 DEBUG   HandlerThread:3912 [handler.py:handle_request():146] handle_request: status_report
2025-03-09 22:21:30,829 ERROR   gpu       :3912 [interfaces.py:monitor():144] Failed to sample metric: Unknown Error
2025-03-09 22:21:30,830 ERROR   gpu       :3912 [interfaces.py:monitor():144] Failed to sample metric: Unknown Error
2025-03-09 22:21:33,244 ERROR   gpu       :3912 [interfaces.py:monitor():144] Failed to sample metric: Unknown Error
2025-03-09 22:21:33,245 ERROR   gpu       :3912 [interfaces.py:monitor():144] Failed to sample metric: Unknown Error
2025-03-09 22:21:34,247 DEBUG   HandlerThread:3912 [handler.py:handle_request():146] handle_request: status_report
2025-03-09 22:21:35,665 ERROR   gpu       :3912 [interfaces.py:monitor():144] Failed to sample metric: Unknown Error
2025-03-09 22:21:35,665 ERROR   gpu       :3912 [interfaces.py:monitor():144] Failed to sample metric: Unknown Error
2025-03-09 22:21:36,295 DEBUG   SenderThread:3912 [sender.py:send():382] send: stats
2025-03-09 22:21:36,783 DEBUG   HandlerThread:3912 [handler.py:handle_request():146] handle_request: stop_status
2025-03-09 22:21:36,784 DEBUG   SenderThread:3912 [sender.py:send_request():409] send_request: stop_status
2025-03-09 22:21:36,798 DEBUG   HandlerThread:3912 [handler.py:handle_request():146] handle_request: internal_messages
2025-03-09 22:21:38,101 ERROR   gpu       :3912 [interfaces.py:monitor():144] Failed to sample metric: Unknown Error
2025-03-09 22:21:38,101 ERROR   gpu       :3912 [interfaces.py:monitor():144] Failed to sample metric: Unknown Error
2025-03-09 22:21:39,431 DEBUG   HandlerThread:3912 [handler.py:handle_request():146] handle_request: status_report
2025-03-09 22:21:40,521 ERROR   gpu       :3912 [interfaces.py:monitor():144] Failed to sample metric: Unknown Error
2025-03-09 22:21:40,522 ERROR   gpu       :3912 [interfaces.py:monitor():144] Failed to sample metric: Unknown Error
2025-03-09 22:21:42,939 ERROR   gpu       :3912 [interfaces.py:monitor():144] Failed to sample metric: Unknown Error
2025-03-09 22:21:42,939 ERROR   gpu       :3912 [interfaces.py:monitor():144] Failed to sample metric: Unknown Error
2025-03-09 22:21:44,481 DEBUG   HandlerThread:3912 [handler.py:handle_request():146] handle_request: status_report
2025-03-09 22:21:45,350 ERROR   gpu       :3912 [interfaces.py:monitor():144] Failed to sample metric: Unknown Error
2025-03-09 22:21:45,351 ERROR   gpu       :3912 [interfaces.py:monitor():144] Failed to sample metric: Unknown Error
2025-03-09 22:21:47,829 ERROR   gpu       :3912 [interfaces.py:monitor():144] Failed to sample metric: Unknown Error
2025-03-09 22:21:47,830 ERROR   gpu       :3912 [interfaces.py:monitor():144] Failed to sample metric: Unknown Error
2025-03-09 22:21:49,535 DEBUG   HandlerThread:3912 [handler.py:handle_request():146] handle_request: status_report
2025-03-09 22:21:50,285 ERROR   gpu       :3912 [interfaces.py:monitor():144] Failed to sample metric: Unknown Error
2025-03-09 22:21:50,286 ERROR   gpu       :3912 [interfaces.py:monitor():144] Failed to sample metric: Unknown Error
2025-03-09 22:21:51,797 DEBUG   HandlerThread:3912 [handler.py:handle_request():146] handle_request: stop_status
2025-03-09 22:21:51,798 DEBUG   SenderThread:3912 [sender.py:send_request():409] send_request: stop_status
2025-03-09 22:21:51,811 DEBUG   HandlerThread:3912 [handler.py:handle_request():146] handle_request: internal_messages
2025-03-09 22:21:55,164 DEBUG   HandlerThread:3912 [handler.py:handle_request():146] handle_request: status_report
2025-03-09 22:21:55,288 ERROR   gpu       :3912 [interfaces.py:monitor():144] Failed to sample metric: Unknown Error
2025-03-09 22:21:55,288 ERROR   gpu       :3912 [interfaces.py:monitor():144] Failed to sample metric: Unknown Error
2025-03-09 22:21:57,715 ERROR   gpu       :3912 [interfaces.py:monitor():144] Failed to sample metric: Unknown Error
2025-03-09 22:21:57,716 ERROR   gpu       :3912 [interfaces.py:monitor():144] Failed to sample metric: Unknown Error
2025-03-09 22:22:00,207 DEBUG   HandlerThread:3912 [handler.py:handle_request():146] handle_request: status_report
2025-03-09 22:22:00,208 ERROR   gpu       :3912 [interfaces.py:monitor():144] Failed to sample metric: Unknown Error
2025-03-09 22:22:00,209 ERROR   gpu       :3912 [interfaces.py:monitor():144] Failed to sample metric: Unknown Error
2025-03-09 22:22:02,645 ERROR   gpu       :3912 [interfaces.py:monitor():144] Failed to sample metric: Unknown Error
2025-03-09 22:22:02,646 ERROR   gpu       :3912 [interfaces.py:monitor():144] Failed to sample metric: Unknown Error
2025-03-09 22:22:05,060 ERROR   gpu       :3912 [interfaces.py:monitor():144] Failed to sample metric: Unknown Error
2025-03-09 22:22:05,061 ERROR   gpu       :3912 [interfaces.py:monitor():144] Failed to sample metric: Unknown Error
2025-03-09 22:22:05,264 DEBUG   HandlerThread:3912 [handler.py:handle_request():146] handle_request: status_report
2025-03-09 22:22:06,297 DEBUG   SenderThread:3912 [sender.py:send():382] send: stats
2025-03-09 22:22:06,810 DEBUG   HandlerThread:3912 [handler.py:handle_request():146] handle_request: stop_status
2025-03-09 22:22:06,811 DEBUG   SenderThread:3912 [sender.py:send_request():409] send_request: stop_status
2025-03-09 22:22:06,825 DEBUG   HandlerThread:3912 [handler.py:handle_request():146] handle_request: internal_messages
2025-03-09 22:22:07,512 ERROR   gpu       :3912 [interfaces.py:monitor():144] Failed to sample metric: Unknown Error
2025-03-09 22:22:07,513 ERROR   gpu       :3912 [interfaces.py:monitor():144] Failed to sample metric: Unknown Error
2025-03-09 22:22:09,935 ERROR   gpu       :3912 [interfaces.py:monitor():144] Failed to sample metric: Unknown Error
2025-03-09 22:22:09,936 ERROR   gpu       :3912 [interfaces.py:monitor():144] Failed to sample metric: Unknown Error
2025-03-09 22:22:11,196 DEBUG   HandlerThread:3912 [handler.py:handle_request():146] handle_request: status_report
2025-03-09 22:22:12,360 ERROR   gpu       :3912 [interfaces.py:monitor():144] Failed to sample metric: Unknown Error
2025-03-09 22:22:12,361 ERROR   gpu       :3912 [interfaces.py:monitor():144] Failed to sample metric: Unknown Error
2025-03-09 22:22:14,787 ERROR   gpu       :3912 [interfaces.py:monitor():144] Failed to sample metric: Unknown Error
2025-03-09 22:22:14,788 ERROR   gpu       :3912 [interfaces.py:monitor():144] Failed to sample metric: Unknown Error
2025-03-09 22:22:16,229 DEBUG   HandlerThread:3912 [handler.py:handle_request():146] handle_request: status_report
2025-03-09 22:22:17,214 ERROR   gpu       :3912 [interfaces.py:monitor():144] Failed to sample metric: Unknown Error
2025-03-09 22:22:17,215 ERROR   gpu       :3912 [interfaces.py:monitor():144] Failed to sample metric: Unknown Error
2025-03-09 22:22:19,621 ERROR   gpu       :3912 [interfaces.py:monitor():144] Failed to sample metric: Unknown Error
2025-03-09 22:22:19,622 ERROR   gpu       :3912 [interfaces.py:monitor():144] Failed to sample metric: Unknown Error
2025-03-09 22:22:21,254 DEBUG   HandlerThread:3912 [handler.py:handle_request():146] handle_request: status_report
2025-03-09 22:22:21,829 DEBUG   HandlerThread:3912 [handler.py:handle_request():146] handle_request: stop_status
2025-03-09 22:22:21,830 DEBUG   HandlerThread:3912 [handler.py:handle_request():146] handle_request: internal_messages
2025-03-09 22:22:21,831 DEBUG   SenderThread:3912 [sender.py:send_request():409] send_request: stop_status
2025-03-09 22:22:22,065 ERROR   gpu       :3912 [interfaces.py:monitor():144] Failed to sample metric: Unknown Error
2025-03-09 22:22:22,065 ERROR   gpu       :3912 [interfaces.py:monitor():144] Failed to sample metric: Unknown Error
2025-03-09 22:22:24,497 ERROR   gpu       :3912 [interfaces.py:monitor():144] Failed to sample metric: Unknown Error
2025-03-09 22:22:24,498 ERROR   gpu       :3912 [interfaces.py:monitor():144] Failed to sample metric: Unknown Error
2025-03-09 22:22:26,926 ERROR   gpu       :3912 [interfaces.py:monitor():144] Failed to sample metric: Unknown Error
2025-03-09 22:22:26,927 ERROR   gpu       :3912 [interfaces.py:monitor():144] Failed to sample metric: Unknown Error
2025-03-09 22:22:27,192 DEBUG   HandlerThread:3912 [handler.py:handle_request():146] handle_request: status_report
2025-03-09 22:22:27,200 DEBUG   HandlerThread:3912 [handler.py:handle_request():146] handle_request: partial_history
2025-03-09 22:22:27,201 DEBUG   SenderThread:3912 [sender.py:send():382] send: history
2025-03-09 22:22:27,202 DEBUG   SenderThread:3912 [sender.py:send_request():409] send_request: summary_record
2025-03-09 22:22:27,205 INFO    SenderThread:3912 [sender.py:_save_file():1392] saving file wandb-summary.json with policy end
2025-03-09 22:22:27,429 DEBUG   HandlerThread:3912 [handler.py:handle_request():146] handle_request: partial_history
2025-03-09 22:22:27,431 DEBUG   SenderThread:3912 [sender.py:send():382] send: history
2025-03-09 22:22:27,432 DEBUG   SenderThread:3912 [sender.py:send_request():409] send_request: summary_record
2025-03-09 22:22:27,433 INFO    SenderThread:3912 [sender.py:_save_file():1392] saving file wandb-summary.json with policy end
2025-03-09 22:22:27,442 DEBUG   SenderThread:3912 [sender.py:send():382] send: telemetry
2025-03-09 22:22:27,480 DEBUG   SenderThread:3912 [sender.py:send():382] send: exit
2025-03-09 22:22:27,480 INFO    SenderThread:3912 [sender.py:send_exit():589] handling exit code: 0
2025-03-09 22:22:27,495 INFO    SenderThread:3912 [sender.py:send_exit():591] handling runtime: 292
2025-03-09 22:22:27,496 INFO    SenderThread:3912 [sender.py:_save_file():1392] saving file wandb-summary.json with policy end
2025-03-09 22:22:27,497 INFO    SenderThread:3912 [sender.py:send_exit():597] send defer
2025-03-09 22:22:27,504 DEBUG   HandlerThread:3912 [handler.py:handle_request():146] handle_request: defer
2025-03-09 22:22:27,504 INFO    HandlerThread:3912 [handler.py:handle_request_defer():172] handle defer: 0
2025-03-09 22:22:27,505 DEBUG   SenderThread:3912 [sender.py:send_request():409] send_request: defer
2025-03-09 22:22:27,506 INFO    SenderThread:3912 [sender.py:send_request_defer():613] handle sender defer: 0
2025-03-09 22:22:27,506 INFO    SenderThread:3912 [sender.py:transition_state():617] send defer: 1
2025-03-09 22:22:27,507 DEBUG   HandlerThread:3912 [handler.py:handle_request():146] handle_request: defer
2025-03-09 22:22:27,507 INFO    HandlerThread:3912 [handler.py:handle_request_defer():172] handle defer: 1
2025-03-09 22:22:27,508 DEBUG   SenderThread:3912 [sender.py:send_request():409] send_request: defer
2025-03-09 22:22:27,508 INFO    SenderThread:3912 [sender.py:send_request_defer():613] handle sender defer: 1
2025-03-09 22:22:27,508 INFO    SenderThread:3912 [sender.py:transition_state():617] send defer: 2
2025-03-09 22:22:27,508 DEBUG   HandlerThread:3912 [handler.py:handle_request():146] handle_request: defer
2025-03-09 22:22:27,509 INFO    HandlerThread:3912 [handler.py:handle_request_defer():172] handle defer: 2
2025-03-09 22:22:27,509 INFO    HandlerThread:3912 [system_monitor.py:finish():203] Stopping system monitor
2025-03-09 22:22:27,509 DEBUG   SystemMonitor:3912 [system_monitor.py:_start():179] Finished system metrics aggregation loop
2025-03-09 22:22:27,528 INFO    HandlerThread:3912 [interfaces.py:finish():202] Joined cpu monitor
2025-03-09 22:22:27,528 DEBUG   SystemMonitor:3912 [system_monitor.py:_start():183] Publishing last batch of metrics
2025-03-09 22:22:27,549 INFO    HandlerThread:3912 [interfaces.py:finish():202] Joined disk monitor
2025-03-09 22:22:27,644 INFO    HandlerThread:3912 [interfaces.py:finish():202] Joined gpu monitor
2025-03-09 22:22:27,645 INFO    HandlerThread:3912 [interfaces.py:finish():202] Joined memory monitor
2025-03-09 22:22:27,645 INFO    HandlerThread:3912 [interfaces.py:finish():202] Joined network monitor
2025-03-09 22:22:27,646 DEBUG   SenderThread:3912 [sender.py:send_request():409] send_request: defer
2025-03-09 22:22:27,647 INFO    SenderThread:3912 [sender.py:send_request_defer():613] handle sender defer: 2
2025-03-09 22:22:27,647 INFO    SenderThread:3912 [sender.py:transition_state():617] send defer: 3
2025-03-09 22:22:27,647 DEBUG   SenderThread:3912 [sender.py:send():382] send: stats
2025-03-09 22:22:27,647 DEBUG   HandlerThread:3912 [handler.py:handle_request():146] handle_request: defer
2025-03-09 22:22:27,648 INFO    HandlerThread:3912 [handler.py:handle_request_defer():172] handle defer: 3
2025-03-09 22:22:27,649 DEBUG   SenderThread:3912 [sender.py:send_request():409] send_request: defer
2025-03-09 22:22:27,649 INFO    SenderThread:3912 [sender.py:send_request_defer():613] handle sender defer: 3
2025-03-09 22:22:27,649 INFO    SenderThread:3912 [sender.py:transition_state():617] send defer: 4
2025-03-09 22:22:27,650 DEBUG   HandlerThread:3912 [handler.py:handle_request():146] handle_request: defer
2025-03-09 22:22:27,650 INFO    HandlerThread:3912 [handler.py:handle_request_defer():172] handle defer: 4
2025-03-09 22:22:27,651 DEBUG   SenderThread:3912 [sender.py:send_request():409] send_request: defer
2025-03-09 22:22:27,651 INFO    SenderThread:3912 [sender.py:send_request_defer():613] handle sender defer: 4
2025-03-09 22:22:27,651 INFO    SenderThread:3912 [sender.py:transition_state():617] send defer: 5
2025-03-09 22:22:27,652 DEBUG   HandlerThread:3912 [handler.py:handle_request():146] handle_request: defer
2025-03-09 22:22:27,652 INFO    HandlerThread:3912 [handler.py:handle_request_defer():172] handle defer: 5
2025-03-09 22:22:27,653 DEBUG   SenderThread:3912 [sender.py:send():382] send: summary
2025-03-09 22:22:27,656 INFO    SenderThread:3912 [sender.py:_save_file():1392] saving file wandb-summary.json with policy end
2025-03-09 22:22:27,657 DEBUG   SenderThread:3912 [sender.py:send_request():409] send_request: defer
2025-03-09 22:22:27,657 INFO    SenderThread:3912 [sender.py:send_request_defer():613] handle sender defer: 5
2025-03-09 22:22:27,657 INFO    SenderThread:3912 [sender.py:transition_state():617] send defer: 6
2025-03-09 22:22:27,657 DEBUG   HandlerThread:3912 [handler.py:handle_request():146] handle_request: defer
2025-03-09 22:22:27,658 INFO    HandlerThread:3912 [handler.py:handle_request_defer():172] handle defer: 6
2025-03-09 22:22:27,658 DEBUG   SenderThread:3912 [sender.py:send_request():409] send_request: defer
2025-03-09 22:22:27,658 INFO    SenderThread:3912 [sender.py:send_request_defer():613] handle sender defer: 6
2025-03-09 22:22:27,674 DEBUG   HandlerThread:3912 [handler.py:handle_request():146] handle_request: status_report
2025-03-09 22:22:27,926 INFO    Thread-16 :3912 [dir_watcher.py:_on_file_modified():288] file/dir modified: c:\Users\D. Sai Mani Kumar\Downloads\DL\da6401_assignment1\wandb\run-20250309_221730-el3kt5nf\files\wandb-summary.json
2025-03-09 22:22:28,007 INFO    SenderThread:3912 [sender.py:transition_state():617] send defer: 7
2025-03-09 22:22:28,008 DEBUG   HandlerThread:3912 [handler.py:handle_request():146] handle_request: defer
2025-03-09 22:22:28,008 INFO    HandlerThread:3912 [handler.py:handle_request_defer():172] handle defer: 7
2025-03-09 22:22:28,009 DEBUG   SenderThread:3912 [sender.py:send_request():409] send_request: defer
2025-03-09 22:22:28,009 INFO    SenderThread:3912 [sender.py:send_request_defer():613] handle sender defer: 7
2025-03-09 22:22:28,489 DEBUG   HandlerThread:3912 [handler.py:handle_request():146] handle_request: poll_exit
2025-03-09 22:22:28,943 INFO    Thread-16 :3912 [dir_watcher.py:_on_file_modified():288] file/dir modified: c:\Users\D. Sai Mani Kumar\Downloads\DL\da6401_assignment1\wandb\run-20250309_221730-el3kt5nf\files\config.yaml
2025-03-09 22:22:29,288 INFO    SenderThread:3912 [sender.py:transition_state():617] send defer: 8
2025-03-09 22:22:29,289 DEBUG   SenderThread:3912 [sender.py:send_request():409] send_request: poll_exit
2025-03-09 22:22:29,289 DEBUG   HandlerThread:3912 [handler.py:handle_request():146] handle_request: defer
2025-03-09 22:22:29,294 INFO    HandlerThread:3912 [handler.py:handle_request_defer():172] handle defer: 8
2025-03-09 22:22:29,295 DEBUG   SenderThread:3912 [sender.py:send_request():409] send_request: defer
2025-03-09 22:22:29,295 INFO    SenderThread:3912 [sender.py:send_request_defer():613] handle sender defer: 8
2025-03-09 22:22:29,295 INFO    SenderThread:3912 [sender.py:transition_state():617] send defer: 9
2025-03-09 22:22:29,296 DEBUG   HandlerThread:3912 [handler.py:handle_request():146] handle_request: defer
2025-03-09 22:22:29,296 INFO    HandlerThread:3912 [handler.py:handle_request_defer():172] handle defer: 9
2025-03-09 22:22:29,297 DEBUG   SenderThread:3912 [sender.py:send_request():409] send_request: defer
2025-03-09 22:22:29,297 INFO    SenderThread:3912 [sender.py:send_request_defer():613] handle sender defer: 9
2025-03-09 22:22:29,297 INFO    SenderThread:3912 [dir_watcher.py:finish():358] shutting down directory watcher
2025-03-09 22:22:29,490 DEBUG   HandlerThread:3912 [handler.py:handle_request():146] handle_request: poll_exit
2025-03-09 22:22:29,955 INFO    SenderThread:3912 [dir_watcher.py:_on_file_modified():288] file/dir modified: c:\Users\D. Sai Mani Kumar\Downloads\DL\da6401_assignment1\wandb\run-20250309_221730-el3kt5nf\files\output.log
2025-03-09 22:22:29,957 INFO    SenderThread:3912 [dir_watcher.py:finish():388] scan: c:\Users\D. Sai Mani Kumar\Downloads\DL\da6401_assignment1\wandb\run-20250309_221730-el3kt5nf\files
2025-03-09 22:22:29,959 INFO    SenderThread:3912 [dir_watcher.py:finish():402] scan save: c:\Users\D. Sai Mani Kumar\Downloads\DL\da6401_assignment1\wandb\run-20250309_221730-el3kt5nf\files\config.yaml config.yaml
2025-03-09 22:22:29,961 INFO    SenderThread:3912 [dir_watcher.py:finish():402] scan save: c:\Users\D. Sai Mani Kumar\Downloads\DL\da6401_assignment1\wandb\run-20250309_221730-el3kt5nf\files\output.log output.log
2025-03-09 22:22:29,970 INFO    SenderThread:3912 [dir_watcher.py:finish():402] scan save: c:\Users\D. Sai Mani Kumar\Downloads\DL\da6401_assignment1\wandb\run-20250309_221730-el3kt5nf\files\requirements.txt requirements.txt
2025-03-09 22:22:29,974 INFO    SenderThread:3912 [dir_watcher.py:finish():402] scan save: c:\Users\D. Sai Mani Kumar\Downloads\DL\da6401_assignment1\wandb\run-20250309_221730-el3kt5nf\files\wandb-metadata.json wandb-metadata.json
2025-03-09 22:22:29,980 INFO    SenderThread:3912 [dir_watcher.py:finish():402] scan save: c:\Users\D. Sai Mani Kumar\Downloads\DL\da6401_assignment1\wandb\run-20250309_221730-el3kt5nf\files\wandb-summary.json wandb-summary.json
2025-03-09 22:22:29,988 INFO    SenderThread:3912 [sender.py:transition_state():617] send defer: 10
2025-03-09 22:22:29,989 DEBUG   SenderThread:3912 [sender.py:send_request():409] send_request: poll_exit
2025-03-09 22:22:29,990 DEBUG   HandlerThread:3912 [handler.py:handle_request():146] handle_request: defer
2025-03-09 22:22:29,991 INFO    HandlerThread:3912 [handler.py:handle_request_defer():172] handle defer: 10
2025-03-09 22:22:29,997 DEBUG   SenderThread:3912 [sender.py:send_request():409] send_request: defer
2025-03-09 22:22:29,998 INFO    SenderThread:3912 [sender.py:send_request_defer():613] handle sender defer: 10
2025-03-09 22:22:29,998 INFO    SenderThread:3912 [file_pusher.py:finish():175] shutting down file pusher
2025-03-09 22:22:30,544 DEBUG   HandlerThread:3912 [handler.py:handle_request():146] handle_request: poll_exit
2025-03-09 22:22:30,546 DEBUG   SenderThread:3912 [sender.py:send_request():409] send_request: poll_exit
2025-03-09 22:22:30,850 INFO    wandb-upload_1:3912 [upload_job.py:push():131] Uploaded file c:\Users\D. Sai Mani Kumar\Downloads\DL\da6401_assignment1\wandb\run-20250309_221730-el3kt5nf\files\output.log
2025-03-09 22:22:30,878 INFO    wandb-upload_0:3912 [upload_job.py:push():131] Uploaded file c:\Users\D. Sai Mani Kumar\Downloads\DL\da6401_assignment1\wandb\run-20250309_221730-el3kt5nf\files\config.yaml
2025-03-09 22:22:30,962 INFO    wandb-upload_3:3912 [upload_job.py:push():131] Uploaded file c:\Users\D. Sai Mani Kumar\Downloads\DL\da6401_assignment1\wandb\run-20250309_221730-el3kt5nf\files\wandb-summary.json
2025-03-09 22:22:31,561 DEBUG   HandlerThread:3912 [handler.py:handle_request():146] handle_request: poll_exit
2025-03-09 22:22:31,562 DEBUG   SenderThread:3912 [sender.py:send_request():409] send_request: poll_exit
2025-03-09 22:22:31,895 INFO    wandb-upload_2:3912 [upload_job.py:push():131] Uploaded file c:\Users\D. Sai Mani Kumar\Downloads\DL\da6401_assignment1\wandb\run-20250309_221730-el3kt5nf\files\requirements.txt
2025-03-09 22:22:32,107 INFO    Thread-15 :3912 [sender.py:transition_state():617] send defer: 11
2025-03-09 22:22:32,108 DEBUG   HandlerThread:3912 [handler.py:handle_request():146] handle_request: defer
2025-03-09 22:22:32,108 INFO    HandlerThread:3912 [handler.py:handle_request_defer():172] handle defer: 11
2025-03-09 22:22:32,109 DEBUG   SenderThread:3912 [sender.py:send_request():409] send_request: defer
2025-03-09 22:22:32,109 INFO    SenderThread:3912 [sender.py:send_request_defer():613] handle sender defer: 11
2025-03-09 22:22:32,109 INFO    SenderThread:3912 [file_pusher.py:join():181] waiting for file pusher
2025-03-09 22:22:32,109 INFO    SenderThread:3912 [sender.py:transition_state():617] send defer: 12
2025-03-09 22:22:32,109 DEBUG   HandlerThread:3912 [handler.py:handle_request():146] handle_request: defer
2025-03-09 22:22:32,109 INFO    HandlerThread:3912 [handler.py:handle_request_defer():172] handle defer: 12
2025-03-09 22:22:32,110 DEBUG   SenderThread:3912 [sender.py:send_request():409] send_request: defer
2025-03-09 22:22:32,110 INFO    SenderThread:3912 [sender.py:send_request_defer():613] handle sender defer: 12
2025-03-09 22:22:32,110 INFO    SenderThread:3912 [file_stream.py:finish():595] file stream finish called
2025-03-09 22:22:32,575 DEBUG   HandlerThread:3912 [handler.py:handle_request():146] handle_request: poll_exit
2025-03-09 22:22:33,062 INFO    SenderThread:3912 [file_stream.py:finish():599] file stream finish is done
2025-03-09 22:22:33,063 INFO    SenderThread:3912 [sender.py:transition_state():617] send defer: 13
2025-03-09 22:22:33,064 DEBUG   SenderThread:3912 [sender.py:send_request():409] send_request: poll_exit
2025-03-09 22:22:33,064 DEBUG   HandlerThread:3912 [handler.py:handle_request():146] handle_request: defer
2025-03-09 22:22:33,066 INFO    HandlerThread:3912 [handler.py:handle_request_defer():172] handle defer: 13
2025-03-09 22:22:33,066 DEBUG   HandlerThread:3912 [handler.py:handle_request():146] handle_request: status_report
2025-03-09 22:22:33,068 DEBUG   SenderThread:3912 [sender.py:send_request():409] send_request: defer
2025-03-09 22:22:33,069 INFO    SenderThread:3912 [sender.py:send_request_defer():613] handle sender defer: 13
2025-03-09 22:22:33,073 INFO    SenderThread:3912 [sender.py:transition_state():617] send defer: 14
2025-03-09 22:22:33,075 DEBUG   HandlerThread:3912 [handler.py:handle_request():146] handle_request: defer
2025-03-09 22:22:33,076 DEBUG   SenderThread:3912 [sender.py:send():382] send: final
2025-03-09 22:22:33,076 INFO    HandlerThread:3912 [handler.py:handle_request_defer():172] handle defer: 14
2025-03-09 22:22:33,077 DEBUG   SenderThread:3912 [sender.py:send():382] send: footer
2025-03-09 22:22:33,078 DEBUG   SenderThread:3912 [sender.py:send_request():409] send_request: defer
2025-03-09 22:22:33,078 INFO    SenderThread:3912 [sender.py:send_request_defer():613] handle sender defer: 14
2025-03-09 22:22:33,082 DEBUG   HandlerThread:3912 [handler.py:handle_request():146] handle_request: poll_exit
2025-03-09 22:22:33,083 DEBUG   SenderThread:3912 [sender.py:send_request():409] send_request: poll_exit
2025-03-09 22:22:33,094 DEBUG   HandlerThread:3912 [handler.py:handle_request():146] handle_request: internal_messages
2025-03-09 22:22:33,098 DEBUG   HandlerThread:3912 [handler.py:handle_request():146] handle_request: server_info
2025-03-09 22:22:33,099 DEBUG   SenderThread:3912 [sender.py:send_request():409] send_request: server_info
2025-03-09 22:22:33,109 DEBUG   HandlerThread:3912 [handler.py:handle_request():146] handle_request: get_summary
2025-03-09 22:22:33,110 DEBUG   HandlerThread:3912 [handler.py:handle_request():146] handle_request: sampled_history
2025-03-09 22:22:33,116 DEBUG   HandlerThread:3912 [handler.py:handle_request():146] handle_request: job_info
2025-03-09 22:22:33,396 DEBUG   SenderThread:3912 [sender.py:send_request():409] send_request: job_info
2025-03-09 22:22:33,407 DEBUG   HandlerThread:3912 [handler.py:handle_request():146] handle_request: shutdown
2025-03-09 22:22:33,408 INFO    HandlerThread:3912 [handler.py:finish():866] shutting down handler
2025-03-09 22:22:34,133 INFO    WriterThread:3912 [datastore.py:close():294] close: c:\Users\D. Sai Mani Kumar\Downloads\DL\da6401_assignment1\wandb\run-20250309_221730-el3kt5nf\run-el3kt5nf.wandb
2025-03-09 22:22:34,413 INFO    SenderThread:3912 [sender.py:finish():1548] shutting down sender
2025-03-09 22:22:34,413 INFO    SenderThread:3912 [file_pusher.py:finish():175] shutting down file pusher
2025-03-09 22:22:34,414 INFO    SenderThread:3912 [file_pusher.py:join():181] waiting for file pusher
